timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -104,41 +104,40 @@ def run(self):
                         try:
                             rule_file_content = rule_file.read()
                             parser = sigma_collection.SigmaCollectionParser(
-                            rule_file_content, self.sigma_config, None)
+                                rule_file_content, self.sigma_config, None)
                             parsed_sigma_rules = parser.generate(sigma_backend)
                         except NotImplementedError as exception:
                             logging.error(
-                                'Error generating rule in file {0:s}: {1!s}'.format(
-                                    rule_file_path, exception))
-                            continue
-                        except Exception as exception:
-                            logging.error(
-                                'Error generating rule in file {0:s}: {1!s}'.format(
-                                    rule_file_path, exception))
+                                'Error generating rule in file {0:s}: {1!s}'
+                                .format(rule_file_path, exception))
                             continue
 
                         for sigma_rule in parsed_sigma_rules:
                             try:
                                 simple_counter += 1
-                                # TODO fix that in Sigma hack to get rid of nested stuff
+                                # TODO fix that in Sigma to remove nested stuff
```

Comment:
what does this mean? `to remove nested stuff` what stuff? what nested

Reply:
this nested stuff: https://github.com/google/timesketch/issues/1199#issuecomment-639475885

Reply:
(the adjustment of the comment was only to address too long lines)

Reply:
Adjusted the comment.